src/components/cell/cell.css

```diff
@@ -4,6 +4,10 @@
   border: 1px solid black;
   display: inline-block;
   vertical-align: middle;
+  text-align: center;
+  line-height: 75px;
+  font-size: 20pt;
+  text-transform: uppercase;
```

Comment:
we're probably going to want to clean this up a little bit.  basically, you can make another container and center the container instead of centering the text within this container.  that way, we potentially have to change less if we want to change the tile size (which we will want to).  nice job figuring it out, though.